controllers/actions.github.com/ephemeralrunnerset_controller.go

```diff
@@ -163,6 +166,34 @@ func (r *EphemeralRunnerSetReconciler) Reconcile(ctx context.Context, req ctrl.R
 		"deleting", len(deletingEphemeralRunners),
 	)
 
+	if r.PublishMetrics {
+		githubConfigURL := ephemeralRunnerSet.Spec.EphemeralRunnerSpec.GitHubConfigUrl
+		parsedURL, err := actions.ParseGitHubConfigFromURL(githubConfigURL)
+		if err != nil {
+			log.Error(err, "Github Config URL is invalid", "URL", githubConfigURL)
+			// stop reconciling on this object
+			return ctrl.Result{}, nil
+		}
+
+		owner := parsedURL.Organization
+		if parsedURL.Enterprise != "" {
+			owner = parsedURL.Enterprise
```

Comment:
we might not want to merge Org/Enterprise into Owner, the owner only means Org or User in GitHub system.